### DISCUSSION TITLE
Computation of resolved TKE (tke_res)

### DISCUSSION THREAD

**Discussion by apossner** [ARM-Development/comble-mip#44] [General]
Is there a common specification over which time and spatial scales Reynolds averaging is to be performed to compute resolved TKE, or do we follow our own definitions?

**Comment by twjuliano** [comment]
Hi @apossner,
For the resolved TKE variable, we're asking for 10-min temporal averaging only (i.e., no spatial averaging).
Thanks,
Tim

**Comment by apossner** [comment]
Hi Tim,
my question was more geared towards how we calculate the velocity fluctuations to calculate resolved TKE, i.e. over which spatio-temporal scales do we characterise the mean flow? Do we compute it with respect to the large-scale winds from the sounding, with respect to instentaneous domain mean flow, 10-min averaged domain mean flow, ect. The choice of spatio-temporal averaging to calculate resolved TKE will impact the diagnostic. We can simply do this as we see fit here of course. I was just wondering if there is a particular way that you would like us to diagnose this variable.
Thanks,
Anna

**Comment by twjuliano** [comment]
Hi Anna,
Thanks for explaining further and apologies I didn't catch that the first time around.
We are asking participants to compute the mean values horizontally from instantaneous fields. I've updated the Excel Sheet accordingly for tke_rs as well as the other variables where we request resolved turbulent quantities.
Please let me know if I can clarifying anything else w.r.t. this point.
Thanks,
Tim

**Reply by apossner** [reply]
Great. Thanks, Tim
